### DISCUSSION TITLE
My first question

### DISCUSSION THREAD

**Discussion by profmikegreene** [profmikegreene/sakai-trinity#16] [Q&A] [answered]
Is this discussion space useful to us?

**Comment by profmikegreene** [comment]
This is an answer... but is it answered?

**Reply by curiousmustard** [reply]
Reply reply

**Reply by curiousmustard** [reply]
Do replies ever collapse?

**Reply by curiousmustard** [reply]
What if there's a bunch of replies? (Sorry Michael)

**Reply by curiousmustard** [reply]
So far no...

**Reply by bendte** [reply]
I do have a Github account! But no profile pic yet.

**Comment by profmikegreene** [comment, marked as answer]
That one wasn't the answer, but this one is.

**Reply by curiousmustard** [reply]
does my reply get the green outline as well?

**Reply by curiousmustard** [reply]
yep!

**Reply by bendte** [reply]
Curious about all the reactions available in Github.

**Reply by bendte** [reply]
> Curious about all the reactions available in Github.

I can select all the reactions!

And I like that the explanatory text for thumbs up is "+1".
